cmd/file_kong2kic.go

```diff
@@ -0,0 +1,123 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/kong/deck/kong2kic"
+	"github.com/kong/go-apiops/logbasics"
+	"github.com/kong/go-database-reconciler/pkg/file"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmdKong2KicInputFilename  string
+	cmdKong2KicOutputFilename string
+	cmdKong2KicAPI            string
+	cmdKong2KicOutputFormat   string
+	cmdKong2KicManifestStyle  string
+	CmdKong2KicClassName      string
+	CmdKong2KicV1beta1        bool
+)
+
+// Executes the CLI command "kong2kic"
+func executeKong2Kic(cmd *cobra.Command, _ []string) error {
+	_ = sendAnalytics("file-kong2kic", "", modeLocal)
+	var (
+		outputContent    *file.Content
+		err              error
+		outputFileFormat file.Format
+	)
+
+	if CmdKong2KicV1beta1 {
+		kong2kic.GatewayAPIVersion = "gateway.networking.k8s.io/v1beta1"
+	} else {
+		kong2kic.GatewayAPIVersion = "gateway.networking.k8s.io/v1"
+	}
+	kong2kic.ClassName = CmdKong2KicClassName
+	verbosity, _ := cmd.Flags().GetInt("verbose")
+	logbasics.Initialize(log.LstdFlags, verbosity)
+
+	inputContent, err := file.GetContentFromFiles([]string{cmdKong2KicInputFilename}, false)
+	if err != nil {
+		return fmt.Errorf("failed reding input file '%s'; %w", cmdKong2KicInputFilename, err)
+	}
+
+	outputContent = inputContent.DeepCopy()
+	if strings.ToUpper(cmdKong2KicOutputFormat) == "JSON" &&
```

Comment:
Since these strings are reused more than once, whould we have these `JSON`, `CRD` etc defined as constants?